app/templates/fd-editform-constructor/form-config-panel.hbs

```diff
@@ -0,0 +1,53 @@
+{{#ui-sidebar
+  class="visible very wide right overlay"
+}}
+  <div class="ui inverted segments">
+    <div class="ui segment">
+      <div class="ui large header">
+        {{t "forms.fd-editform-constructor.caption"}}
+      </div>
+    </div>
+    <div class="ui segment">
+      <form class="ui form">
+        <div class="ui small dividing header">
+          {{t "forms.fd-editform-constructor.form-config-panel.form-properties-header"}}
+        </div>
+        <div class="field">
+          <label>{{t "forms.fd-editform-constructor.form-config-panel.form-name"}}</label>
+          {{flexberry-textbox value=model.editform.caption}}
+        </div>
+        <div class="field">
+          <label>{{t "forms.fd-editform-constructor.form-config-panel.form-description"}}</label>
+          {{flexberry-textarea value=model.editform.description}}
+        </div>
+        <div class="ui small dividing header">
+          {{t "forms.fd-editform-constructor.form-config-panel.control-properties-header"}}
+        </div>
+        {{#if _selecetedIsControl}}
+          <div class="field">
+            <label>{{t "forms.fd-editform-constructor.form-config-panel.control-caption"}}</label>
+            {{flexberry-textbox value=selectedControl.caption}}
+          </div>
+          <div class="field">
+            <label>{{t "forms.fd-editform-constructor.form-config-panel.control-type"}}</label>
+            {{flexberry-dropdown items=(array "int" "bool" "string" "date") value=selectedControl.type}}
```

Comment:
I'm not really sure, but maybe it is better to create array in controller?

Reply:
Yes. I do not know yet what it will be, this is for example.